app/services/slack_notifier_service.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "slack-notifier"
+
+class SlackNotifierService
+  include ServicePattern
+
+  def initialize(channel: "#twd_publish_register_tech", message:)
+    @channel = channel
+    @message = message
+    @notifier = Slack::Notifier.new(ENV.fetch("SLACK_WEBHOOK_URL"))
```

Comment:
Traditionally we use `Settings` for this kind of configuration, gives us a unified approach to that. Is there a reason `SLACK_WEBHOOK_URL` shouldn't be in there?

Reply:
Nope, will change!